DirectProgramming/C++SYCL_FPGA/ReferenceDesigns/board_test/README.md

```diff
@@ -5,6 +5,7 @@ The `Board Test` sample is a reference design contains tests to check FPGA board
 - Kernel clock frequency
 - Kernel launch latency
 - Kernel to device global memory bandwidth
+- Unified Shared Memory bandwidth
 
 | Area                    | Description
 |:---                     |:---

```

Comment:
this comment is for line 16:

This reference design implements tests to check FPGA board interfaces and measure host-to-device and kernel-to-global memory interface metrics. ~~Custom platform developers can~~ Use this reference design as a starting point to validate ~~custom~~ platform interfaces **when you customize a BSP**.

Reply:
will be done in separate PR